detection-rules/attachment_any_embedded_b64_pe

```diff
@@ -0,0 +1,13 @@
+name: "Attachment: File with embedded base64-encoded executable"
+description: |
+  Attachment contains a base-64 encoded executable.
+type: "rule"
+severity: "high"
+  type.inbound 
+  and any(attachments,
+    any(beta.binexplode(.), 
+      any(.flavors.yara, . == 'base64_pe')) 
+  )
+  and sender.email.email not in $recipient_emails
```

Comment:
based on how great the performance of this rule is in some Hunts we've run, i think we can actually do without any historical context. i have yet to see any FPs! if that changes, i'll let you know

also check out [this unsolicited snippet](https://docs.sublimesecurity.com/docs/common-snippets#unsolicited-has-my-organization-ever-sent-an-email-to-this-sender), which is a bit more flexible